easybuild/easyconfigs/z/zbar/zbar-0.23.1-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,48 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'zbar'
+version = '0.23.1'
+
+homepage = 'https://github.com/mchehab/zbar'
+description = """ZBar is an open source software suite for reading bar codes
+from various sources, including webcams. As its development stopped in 2012,
+I took the task of keeping it updated with the V4L2 API. This is the main
+repository for it. There's a clone at at LinuxTV.org, and another one at gitlab."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/mariusmuja/flann/archive/']
```

Comment:
This source URL seems to be incorrect, copy-paste error?

Reply:
Erm, yes, copy&paste error. Thanks for spotting that.